### PR TITLE
Build ignition packages with bazel in a Docker container

### PR DESCRIPTION
**Pull request by mjcarroll**

### CODE REVIEW

File: example/bazel.repos [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    url: https://github.com/ignitionrobotics/ign-bazel
+    version: docker
+  ign_common:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-common
+    version: fix_tests
+  ign_math:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-math
+    version: fixup_sonames
```
Comment (JShep1) [inline example/bazel.repos:21]: This can go back to `bazel` branch

File: example/bazel.repos [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    url: https://github.com/osrf/sdformat
+    version: bazel
+  ign_bazel:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-bazel
+    version: docker
+  ign_common:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-common
+    version: fix_tests
```
Comment (JShep1) [inline example/bazel.repos:17]: This will eventually need to be `bazel`

File: example/bazel.repos (hunk 1 above)
Comment (mjcarroll) [reply example/bazel.repos]: Done.

File: example/bazel.repos (hunk 2 above)
Comment (mjcarroll) [reply example/bazel.repos]: I went ahead and did it, I think we can land that pr as well.

**Comment by mjcarroll** [comment]
> is the repos.example file necessary?

Nope.

**Pull request merged**